L1Trigger/CSCTriggerPrimitives/src/CSCGEMMatcher.cc

```diff
@@ -53,33 +55,31 @@ unsigned CSCGEMMatcher::calculateGEMCSCBending(const CSCCLCTDigi& clct, const GE
       else
         slope = lookupTableME21ILT_->es_diff_slope_L2_ME21_odd(diff);
     }
-  }
-
-  const bool isME1a(station_ == 1 and clct.getKeyStrip() > CSCConstants::MAX_HALF_STRIP_ME1B);
-
-  if (station_ == 1 and isME1a) {
-    if (isEven_) {
-      if (cluster.id().layer() == 1)
-        slope = lookupTableME11ILT_->es_diff_slope_L1_ME1a_even(diff);
-      else
-        slope = lookupTableME11ILT_->es_diff_slope_L2_ME1a_even(diff);
-    } else {
-      if (cluster.id().layer() == 1)
-        slope = lookupTableME11ILT_->es_diff_slope_L1_ME1a_odd(diff);
-      else
-        slope = lookupTableME11ILT_->es_diff_slope_L2_ME1a_odd(diff);
-    }
-  } else {
-    if (isEven_) {
-      if (cluster.id().layer() == 1)
-        slope = lookupTableME11ILT_->es_diff_slope_L1_ME1b_even(diff);
-      else
-        slope = lookupTableME11ILT_->es_diff_slope_L2_ME1b_even(diff);
+  } else {  // station == 1
+    if (isME1a) {
```

Comment:
```suggestion
    if ( clct.getKeyStrip() > CSCConstants::MAX_HALF_STRIP_ME1B ) { //is in ME1a
```

Reply:
I suggest moving the ME1a requirement to the sole place in the subfunction it is used explicitly and just comment in what it does without doubling up the station 1 requirement, then.